src/mcode-pilot/services/outcomes/CLQOutcomesService.js

```diff
@@ -11,20 +13,60 @@ export default class CLQOutcomesService extends IOutcomesService {
         this.filters = params.filters;
     }
 
+    _genderMapping(gender) {
+        let lowered = gender ? _.toLower(gender) : null;
+        let code = 'UNK';
+        let display = 'UNKOWN';
+        if (lowered === "female" || lowered === 'f') {
+            code = '703118005';
+            display = 'Female';
+        } else if (lowered === 'male' || lowered === 'm') {
+            code = '703117000';
+            display = 'Male';
+        }
+        return {
+            code: code,
+            displayName: display,
+            codeSystem: 'SNOMEDCT'
+        };
+    }
+
+    __raceCodeMapping(race) {
+        let code = find_race_code(race);
+        if (code) {
+            // if the value is one of the codes from the code system make sure it is one that clq supports
+            if (!['2028-9', '2106-3', '2054-5', '2131-1'].indexOf(code.code)) {
```

Comment:
```suggestion
            if (!['2028-9', '2106-3', '2054-5', '2131-1'].includes(code.code)) {
```
`indexOf` returns `-1` if the given code isn't in the list, which is a truthy value, and `0` if the given code is the first item in the list ('2028-9') which is a falsy value. I think you want `includes` here